content/en/apps/examples/training.md

```diff
@@ -42,19 +42,14 @@ The CHT’s onboarding and training capabilities offer a remote way to provide e
 
 ### Remote Login by App
 
-When creating users, the admin has the option to send a user their credentials via SMS using a [magic link](https://hackernoon.com/magic-links-d680d410f8f7). This generates a new, random and complex password with a 24-hour expiry. If no gateway is set up, the message may be sent via another messaging app. 
+When creating users, the admin has the option to send a user their credentials via SMS using a [magic link](https://hackernoon.com/magic-links-d680d410f8f7). This generates a new, random and complex password with a 24-hour expiry.
```

Comment:
I think it would be better to drop the phrase "magic link" as well as the hyperlink to "hackernoon.com". I don't expect the people reading this to know what that is, and it's sort of an implementation detail. We can just say "link", right?

Reply:
Yep!